products/bun.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Bun
+category: framework
+tags: javascript-runtime
```

Comment:
I see `nodejs` is also tagged with this, but I was considering `javascript-runtime` as products "that run on the JS runtime". 

@marcwrobel thoughts?

Reply:
That's correct, I had this in mind too.

A similar case is Java, for which the various distribution has been flagged with https://endoflife.date/tags/java-distribution. But in this case it is in fact really a JavaScript runtime, so not sure what to do about it. So let's use the same tags as Node.js for now.